src/main/java/moe/nightfall/vic/integratedcircuits/cp/CircuitData.java

```diff
@@ -31,7 +32,7 @@
 public class CircuitData implements Cloneable {
 
 	// cdata version
-	public static final int version = 1;
+	public static final int version = 2;
```

Comment:
Note that the legacy loader doesn't work at all (because it doesn't do the steps in sequence, there's a big TODO on it), so if you decide to change the version it should probably be fixed first